lms/djangoapps/edxnotes/tests.py

```diff
@@ -566,42 +563,41 @@ def test_preprocess_collection_no_item(self):
         """
         initial_collection = [
             {
-                u"quote": u"quote text",
-                u"text": u"text",
-                u"usage_id": text_type(self.html_module_1.location),
-                u"updated": datetime(2014, 11, 19, 8, 5, 16, 00000).isoformat()
+                "quote": "quote text",
+                "text": "text",
+                "usage_id": str(self.html_module_1.location),
+                "updated": datetime(2014, 11, 19, 8, 5, 16, 00000).isoformat()
             },
             {
-                u"quote": u"quote text",
-                u"text": u"text",
-                u"usage_id": text_type(self.course.id.make_usage_key("html", "test_item")),
-                u"updated": datetime(2014, 11, 19, 8, 6, 16, 00000).isoformat()
+                "quote": "quote text",
+                "text": "text",
+                "usage_id": str(self.course.id.make_usage_key("html", "test_item")),
+                "updated": datetime(2014, 11, 19, 8, 6, 16, 00000).isoformat()
             },
         ]
 
-        six.assertCountEqual(
-            self,
+        self.assertCountEqual(
```

Comment:
We've removed `self.assert` checks. You should now use `assert len(expr1) == len(expr2)` instead of this.